hazelcast/src/test/java/com/hazelcast/nio/IOUtilTest.java

```diff
@@ -131,25 +157,27 @@ public void testWriteAndReadByteArray_withNull() throws Exception {
 
     @Test
     public void testWriteAndReadObject() throws Exception {
-        String expected = "test input";
+        InternalSerializationService serializationService = new DefaultSerializationServiceBuilder().build();
```

Comment:
Isn't it easier to have the serializationService as a field on this test? Are there any serializationServices created with different settings?

Reply:
Fixed